src/components/Details/VmDetails/VmDetails.js

```diff
@@ -0,0 +1,197 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { has, isNull } from 'lodash';
+import { VmStatus } from '../../VmStatus';
+import {
+  getCloudInitData,
+  getCpu,
+  getDescription,
+  getFlavor,
+  getMemory,
+  getNodeName,
+  getOperatingSystem,
+  getVmTemplate,
+  getWorkloadProfile,
+} from '../../../utils';
+import { CUSTOM_FLAVOR } from '../../../constants';
+
+const DASHES = '---';
+
+const cloudInitInUse = vm => !isNull(getCloudInitData(vm));
+
+const OnOffReporter = props => {
+  const statusText = props.on ? 'On' : 'Off';
+  const statusIcon = props.on ? 'pficon pficon-on' : 'pficon pficon-off';
+  const statusColor = props.on ? '#3f9c35' : '#a30000';
+
+  return (
+    <span>
+      <span style={{ color: `${statusColor}` }} className={statusIcon} /> {statusText}
+    </span>
+  );
+};
+
+OnOffReporter.propTypes = {
+  on: PropTypes.bool.isRequired,
+};
+
+const VmStatusReporter = props => {
```

Comment:
I find it strange to have this component which is very similar to VmStatus and it is not reporting the same status -> this one has only on/off and VmStatus has a lot more. It results in having two different informations on one page - ON from VmStatusReporter and ie Starting from VmStatus. We can resolve it in a follow up issue - lets talk do UXD.

Reply:
https://docs.google.com/document/d/1Fh2eb9TD0aq9JulVbLlss2ucdqYzx_uqTaThse_fcgE/edit?disco=AAAACHTFmh4

Reply:
@pcbailey we can remove the VmStatusReporter, see the discussion in the link i provided above

Reply:
Done!